packages/graphql/src/utils/execute.ts

```diff
@@ -67,7 +62,60 @@ async function execute(input: {
         }
     }
 
-    const session = input.context.driver.session(sessionParams);
+    return sessionParams;
+};
+
+const getTransactionConfig = () => {
+    const app = `${environment.NPM_PACKAGE_NAME}@${environment.NPM_PACKAGE_VERSION}`;
+    return {
+        metadata: {
+            app,
+            type: "user-transpiled",
+        },
+    };
+}
+
+const getExecutor = (input: {
+    defaultAccessMode: SessionMode;
+    context: Context;
+}): { driver?: Driver; session?: Session; transaction: Transaction } => {
+    if (!input.context.executionContext) {
```

Comment:
Perhaps assign `executionContext` here to avoid constantly doing `input.context.executionContext`?

```suggestion
    const executionContext = input.context.executionContext;
    
    if (!executionContext) {
```